app/controllers/ctc/questions/confirm_payment_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Ctc
+  module Questions
+    class ConfirmPaymentController < QuestionsController
+      include AuthenticatedCtcClientConcern
+
+      layout "intake"
+
+      def edit
+        @ctc_amount = ChildTaxCreditCalculator.total_advance_payment_2021(current_intake.dependents)
+        @rrc_amount = current_intake.tax_return(2020)&.claimed_recovery_rebate_credit || 0
+        @third_stimulus_amount = current_intake.tax_return(2020)&.expected_recovery_rebate_credit_three || 0
+        @not_collecting = (@ctc_amount.zero? && @rrc_amount.zero? && @third_stimulus_amount.zero?) ? true : false
+
+        send_mixpanel_event(event_name: "ctc_efile_estimated_payments", data: {
```

Comment:
I wonder if we can find another place to send this mixpanel event. for example, I'd love if we can tie sending the mixpanel event into some transition of the intake being completed.

We currently run into a ton of errors with sending the same email or text multiple times when we mix side effects into the controller get or post methods like this. sending this data to mixpanel multiple times may affect the quality of the data.

Reply:
Note to ourselves: Does Kelly have the capability to de-dupe these events for unique visitors in mixpanel? Otherwise we could put an event when we queue the submission... or something.

Reply:
Kelly says data science will use this (if anyone) and the events will be connected to a `visitor_id` so data science will be able to de-dupe.